java-checks/src/test/java/org/sonar/java/checks/ImmediateReverseBoxingCheckTest.java

```diff
@@ -21,13 +21,24 @@
 
 import org.junit.jupiter.api.Test;
 import org.sonar.java.checks.verifier.CheckVerifier;
+import org.sonar.java.checks.verifier.TestUtils;
+import org.sonar.java.checks.verifier.internal.InternalCheckVerifier;
 
 class ImmediateReverseBoxingCheckTest {
 
   @Test
   void test() {
+    InternalCheckVerifier.newInstance()
+      .onFile(TestUtils.testSourcesPath("checks/ImmediateReverseBoxingCheck.java"))
```

Comment:
in almost all the other rules, we use static imports for `testSourcesPath` and `nonCompilingTestSourcesPath`. I would suggest to keep doing this!